MetaMask/metamask-extension
# Nft build for e2e tests

## Explanation

Fixes: https://github.com/MetaMask/metamask-extension/issues/17257

This PR creates a new build with NFT's enabled so we can write e2e tests for various features that aren't present in the main build.

A simple test has been add to test the nft build

Parallelism for the nft e2e tests is not necessary as there will only be a handful of tests, therefore i've set this to 1. We can increase this later if needed



## Screenshots/Screencaps



### Before

N/A



### After

Circle ci jobs for nft tests in chrome and firefox

<img width="844" alt="nft e2e tests" src="https://user-images.githubusercontent.com/53189696/214844851-137fc38d-9fed-47f8-a069-ceaeb475465e.png">




## Manual Testing Steps
Check the following build jobs are green:
- ci/circleci: test-e2e-chrome-nft 
- ci/circleci: test-e2e-firefox-nft 



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
